app/models/search_form.rb

```diff
@@ -0,0 +1,35 @@
+class SearchForm
+
+  ADMIN_SEX = %w[A F M N O U]
+
+  attr_reader(*QueryParameters.all)
+
+  # Attribute name must match the key name expected by V2MessageBuilder as
+  # the hash is generated in to_query using attribute names
+  def initialize(params)
+    @patient_name = hash_or_nil(params.slice(:family_name, :given_name))
+    @patient_dob = params[:patient_dob]
+    phone = Phonelib.parse(params[:phone])
+    @phone = hash_or_nil(area_code: phone.area_code, local_number: phone.local_number)
```

Comment:
`local_number` should only be the last 7 digits, and with no spaces or hyphens.

Right when I search for patient [RylandAIRA PawneeAIRA](http://vci.mitre.org:8081/iis-sandbox/patient?patientReportedId=12), the following is being passed into the `query()` function:

`:phone=>{:area_code=>"586", :local_number=>"(586) 412-4271"}`